lib/primitives/array-element.js

```diff
@@ -92,6 +92,15 @@ module.exports = function(BaseElement, registry) {
       return undefined;
     },
 
+    /*
+     * It reduces through a given array and calls get on each item.
+     */
+    getIn: function(paths) {
```

Comment:
Is #getIn the best name? And your description is even more implementation specific vs. function specific here. Something to think about is a more representative name for the function and documentation that describes the end result vs how it does it.